__tests__/app.test.js

```diff
@@ -13,38 +13,92 @@ beforeEach(() => {
+    });
+
+    test("200: all topics are returned", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const { topics } = response.body;
+          expect(topics).toHaveLength(3);
+          topics.forEach((topic) => {
+            expect(topic).toHaveProperty("slug", expect.any(String));
+            expect(topic).toHaveProperty("description", expect.any(String));
+          });
         });
-      });
+    });
   });
-  test("200: returns api documentaion", () => {
-    return request(app)
-      .get("/api")
-      .expect(200)
-      .then((response) => {
-        expect(response.body).toEqual(endpointDocumentation);
-        expect(response.body["GET /api"].description).toEqual(
-          "serves up a json representation of all the available endpoints of the api"
-        );
-        expect(response.body["GET /api/topics"].description).toEqual(
-          "serves an array of all topics"
-        );
-        expect(response.body["GET /api/articles"].description).toEqual(
-          "serves an array of all articles"
-        );
-      });
+
+  describe("GET getArticlesById", () => {
+    test("200: responds with the correct article object on the response body", () => {
+      return request(app)
+        .get("/api/articles/13")
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body;
+          expect(article).toEqual({
+            article_id: 13,
+            title: "Another article about Mitch",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "There will never be enough articles about Mitch!",
+            created_at: "2020-10-11T11:24:00.000Z",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          });
+        });
+    });
+    test("200: for id: 13 responds with the correct article object on the response body", () => {
+      return request(app)
+        .get("/api/articles/13")
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body;
+
+          expect(article).toHaveProperty("article_id", expect.any(Number));
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).toHaveProperty("body", expect.any(String));
+          expect(article).toHaveProperty("votes", expect.any(Number));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).toHaveProperty("created_at", expect.any(String));
+        });
+    });
+    test("404: responds with error when given an unknown article id", () => {
+      return request(app).get("/api/articles/5000").expect(404);
```

Comment:
Since your errors are custom with messages, I would test that the message comes back as expected too